plugin.xml

```diff
@@ -9,21 +9,24 @@ xmlns:android="http://schemas.android.com/apk/res/android"
     <keywords>cordova</keywords>
 
     <engines>
-        <engine name="cordova" version=">=9.0.0"/>
```

Comment:
What's the impact of changing this requirement?

Reply:
I believe that in order for this version to be aligned with the version used my MABS 6.3, we needed to lower this version to 7.1.0.